dask-gateway-dev.yaml

```diff
@@ -0,0 +1,58 @@
+# OPTIONAL conda environment file to help get setup with
+# dependencies to build + test dask-gateway locally
```

Comment:
```suggestion
# This is an optional conda environment file to setup a local development
# environment for this project, to help develop, build, and run tests locally
#
# conda env create --file dev-environment.yaml
#
```

I think the conda environment name is fine, it is a name that in a computer wide context is helpful to understand it is the development environment for this project. But, the filename within this project isn't very helpful. Can we rename it to be dev-environment.yaml instead? These files are typically named `environment.yml` or similar which would indicate it is a conda environment file, and it is a common practice for example to have `dev-requirements.txt` and `requirements.txt` for pip environment files.

So the gist is, by renaming the file, we help clarify the files purpose within this project, based on its file-name without needing to inspect the content of it.

Reply:
This makes sense. Thank you for the recommendation!